docs/react/use-custom-date-library.en-US.md

```diff
@@ -119,11 +119,63 @@ Create `src/components/DatePicker.tsx`.
 Code as follows:
 
 ```tsx
-import dateFnsGenerateConfig from 'rc-picker/lib/generate/dateFns';
 import generatePicker from 'antd/es/date-picker/generatePicker';
 import 'antd/es/date-picker/style/index';
+import dateFnsGenerateConfig from 'rc-picker/lib/generate/dateFns';
 
 const DatePicker = generatePicker<Date>(dateFnsGenerateConfig);
 
 export default DatePicker;
 ```
+
+## Use luxon
+
+[luxon](https://moment.github.io/luxon/) can be used instead of `dayjs` and supports the same functionality, but it does introduce some differences in behavior that we will explain below.
```

Comment:
```suggestion
Since `antd 5.4.0`, [luxon](https://moment.github.io/luxon/) can be used instead of `dayjs` and supports the same functionality, but it does introduce some differences in behavior that we will explain below.
```

Reply:
Thank you 🙏 Adjusted in this commit: b9e00ebb8f800723c3da2e02e40b1adace1925c5